core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -14,6 +14,8 @@ package kafka.api
 
 import java.nio.ByteBuffer
 import java.util
+import java.time.Duration
```

Comment:
nit: unneeded import. A couple more in `ConsumerBounceTest` and `PlaintextConsumerTest`.